src/transformers/utils/hub.py

```diff
@@ -588,6 +541,7 @@ def get_file_from_repo(
         subfolder=subfolder,
         _raise_exceptions_for_missing_entries=False,
         _raise_exceptions_for_connection_errors=False,
+        use_auth_token=deprecated_kwargs.pop("use_auth_token", None),
```

Comment:
Despite we also give warning in `cached_file`, the goal is not to use `use_auth_token` anywhere. If there is no important reason for the change inside this method, I would keep it as it is.

Reply:
The only reason was to avoid code duplication (yet keeping the warning message). But ok to revert it since `get_file_from_repo` will be removed soon anyway.
Reverted in https://github.com/huggingface/transformers/pull/27564/commits/bdc27689dc069057751db7fc3de36731760a4ece.